app/views/membership_applications/show.html.haml

```diff
@@ -9,12 +9,8 @@
     %p= @membership_application.phone_number
   %p= mail_to @membership_application.company_email, @membership_application.company_email
   .wpcf7
-    = form_for @membership_application, url: membership_application_path do |f|
-      = f.select(:status, [['Pending', 'Pending'],
-                           ['Accepted', 'Accepted'],
-                           ['Rejected', 'Rejected']])
-      = f.submit 'Update', class: 'wpcf7-submit'
-      - if @membership_application.status == 'Accepted'
-        %p Membership accepted at #{@membership_application.updated_at.strftime('%F')}
-      - if @membership_application.status == 'Rejected'
-        %p Membership rejected at #{@membership_application.updated_at.strftime('%F')}
\ No newline at end of file
+    Application Status:
+    - if policy(@membership_application).permitted_attributes_for_edit.include? :status
```

Comment:
It feels like a bit too much logic for a view but IF  there's no other way to do this I'm okay with it. Extract this to a helper perhaps?